### ISSUE TITLE
[Bug]: On a "new transaction" mobile page clicking "payee" takes two clicks instead of one 

### ISSUE THREAD

**Issue by minajevs** [actualbudget/actual#3268]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue
- [X] I will be providing steps how to reproduce the bug (in most cases this will also mean uploading a demo budget file)

### What happened?

When on mobile, on a "new transaction" page clicking on any form field (payee, category, account, date, notes) takes 2 clicks to open the field instead of 1. 

Looks like amount field is focused by default, and then first click outside unfocuses the amount field, and requires second click to focus another element. Instead it should immediately switch focus to another clicked element.

See recording:
https://github.com/user-attachments/assets/fbe9d80b-4382-466d-846f-5ca6d24c1330

### Where are you hosting Actual?

Pikapods

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari

### Operating System

Mobile Device

**Comment by minajevs** [comment]
Can I try to fix the issue myself? I am interested in contributing :)

**Comment by youngcw** [comment]
> Can I try to fix the issue myself? I am interested in contributing :)

Go ahead if you want to try

**Comment by DomiJV** [comment]
I have the same problem and i think it is the same issue as described here:
https://github.com/actualbudget/actual/issues/1735

**Comment by MatissJanis** [comment]
👋  This might be fixed already in one of the latest versions. Would you mind verifying @minajevs ?

**Comment by minajevs** [comment]
> 👋 This might be fixed already in one of the latest versions. Would you mind verifying [@minajevs](https://github.com/minajevs) ?

Just tested on https://demo.actualbudget.org/transactions/new and it is still actual (hehe). 

My PR, unfortunately, was closed as stale without a review, and quickly going thought the updates to the code in the latest releases, there doesn't seem to be any related fixes. 

I have also decided not to migrate to Actual since then, so I doubt I will have time/energy to refresh the PR. Anyone is welcome to pick it up and try again.

**Comment by wtsiamruk** [comment]
yeah i was about to report this one, along with some enrypted file reset sync issue (it's not downloading if you hit revert as per doc:

https://actualbudget.org/docs/getting-started/sync/#syncing-has-been-reset-on-this-cloud-file

i'll try to come with 2 PRs for these 2 issues.....

**Comment by wtsiamruk** [comment]
i think this one is actually fixed in last version

**Comment by wtsiamruk** [comment]
https://github.com/user-attachments/assets/9d6cd62a-25cb-4943-a5e2-0e7b3511215b

**Comment by wtsiamruk** [comment]
@MatissJanis it can be closed
